package.json

```diff
@@ -29,6 +29,7 @@
     "moment": "~2.24.0",
     "mout": "~1.1.0",
     "needle": "~2.1.2",
+    "nock": "~12.0.3",
```

Comment:
there a reason this is an actual dependency now?

Reply:
Good catch; I thought I fixed that 🤔